muesli/web/templates/lecture/edit.pt

```diff
@@ -103,7 +103,7 @@ Zu dieser ${names['name']} werden die folgenden ${names['tutorials']} angeboten:
 <p><a tal:attributes="href request.route_path('lecture_add_exam', lecture_id=lecture.id)">Testat hinzufügen</a></p>
 
 <div tal:repeat="category categories" tal:omit-tag="">
-  <h4 tal:condition="exams[category['id']].count()!=0">${category['name']}</h4>
+  <h4 tal:condition="len(exams[category['id']])!=0">${category['name']}</h4>
```

Comment:
Hallo @ekzyis, 
was im Müsli noch an vielen Stellen falsch benutzt wird ist hier ein Codeschnipsel `len(...) != 0`, in Python gibt es die Möglichkeit die implizit boolschen Werte eines Containers zu nutzen, das ist aber eher Coding style und kommt im Müsli auch oft vor :smile: 
In diesem fall würde das dann so aussehen:

```
<h4 tal:condition="exams[category['id']]">${category['name']}</h4>
```
_Mehr Infos dazu findest du [hier](https://anh.cs.luc.edu/python/hands-on/3.1/handsonHtml/boolean.html)._

(Lass dir von mir übrigens nichts sagen, verantwortlich für das Müsli ist @tynsh)

Reply:
Ach ja, stimmt, daran hatte ich gar nicht gedacht.

Mit dem expliziten Vergleich ist es einerseits minimal lesbarer, aber andererseits auch irgendwie umständlich ausführlich geschrieben :D

Danke für deine Antwort :+1: